iphone/Classes/TiUIListView.m

```diff
@@ -1762,7 +1762,7 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
     }
     if (TiDimensionIsDip(height)) {
         return height.value;
-    } else if (TiDimensionIsAutoSize(height)) {
+    } else if (TiDimensionIsAutoSize(height) || TiDimensionIsUndefined(height)) {
```

Comment:
The test-case works for me. But I am wondering if we shouldn't include this as well:
```objc
    } else if (TiDimensionIsAuto(height) || TiDimensionIsAutoSize(height) || TiDimensionIsUndefined(height)) {
```
And I also noticed that the template do has a height and width specified, so it shouldn't go until that line, can you check that before approving? Thx! Good work!